docs/source/tutorial/schema.mdx

```diff
@@ -168,12 +168,12 @@ Now that we have scoped out our app's schema, let's run the server by calling `s
 _src/index.js_
```

Comment:
The following code block was converted to TypeScript, but this filename and the codeblock language tag stayed the same. These should probably be changed, and maybe we should wrap the codeblock in a `MultiCodeBlock`.

Reply:
Alternatively, we can just return this codeblock to its original JS format, since we're not converting the server part of the tutorial to TS yet!